app/models/manageiq/providers/ibm_terraform/provider.rb

```diff
@@ -23,6 +23,8 @@ class ManageIQ::Providers::IbmTerraform::Provider < ::Provider
   validates :url,          :presence => true
   validates :identity_url, :presence => true
 
+  supports :create
```

Comment:
I think you want this on the `IbmTerraform::ConfigurationManager` not the `Provider`